var/spack/repos/builtin/packages/nfft/package.py

```diff
@@ -6,64 +6,67 @@
 from spack import *
 
 
-class Nfft(AutotoolsPackage):
-    """NFFT is a C subroutine library for computing the nonequispaced discrete
-    Fourier transform (NDFT) in one or more dimensions, of arbitrary input
-    size, and of complex data."""
+class NfftBase(AutotoolsPackage):
```

Comment:
Why is a base class needed? Do you plan to have other packages inheriting from it?

Reply:
Thx!  (as I said: playing with matches! :) ) I presume that design is used in the FFTW package to deal with simultaneous multiple precision builds, ... not  sure